docs/build/isc/v1.3-alpha/docs/how-tos/core-contracts/token/send-token-across-chains.mdx

```diff
@@ -0,0 +1,108 @@
+---
+description: How to Send Native Tokens Across Chains.
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - EVM
+  - how-to
+  - native tokens
+  - cross-chain
+---
+import ExampleCodeIntro from '../../../_partials/how-tos/token/_example_code_intro.md';
+import DemoTokenSetup from '../../../_admonitions/_token-demo-setup.md';
+import CreateNativeToken from '../../../_admonitions/_create-native-token.md';
+
+# Send Native Token Across Chains
+
+In this guide, you'll learn how to send L1 native Tokens from L1 to an L2 EVM account on a destination chain by using the `sendCrossChain` function of the `NativeTokenController.sol` contract.
+
+<CreateNativeToken/>
+
+## Understanding the `sendCrossChain` Function
+
+First, let’s take a look at the `sendCrossChain` function in the `NativeTokenController.sol` file:
+
+```solidity
+function sendCrossChain(
+        address destinationChain,
+        bytes memory destinationAddress,
+        uint256 amount,
+        ISCChainID chainID,
+        uint64 storageDeposit
+) external {
+    // Function implementation
+}
+```
+
+This function facilitates the transfer of a specified `amount` of tokens from the current chain to a `destinationChain`, with the `destinationAddress` as the recipient on the destination chain. The `sendCrossChain` function invokes the `ISC.sandbox.send` function, which manages the actual cross-chain message transmission.
```

Comment:
Does it make sense to include the original function doc comments? 
Also, I think the term `wrapper function` is more descriptive, WDYT?

Reply:
I modified the paragraph to implement your suggestions.